ext/bcmath/bcmath.c

```diff
@@ -777,4 +791,938 @@ PHP_FUNCTION(bcscale)
 /* }}} */
 
 
+
+static zend_class_entry *bcmath_number_ce;
+static zend_object_handlers bcmath_number_obj_handlers;
+
+static zend_always_inline bcmath_number_obj_t *get_bcmath_number_from_obj(const zend_object *obj)
+{
+	return (bcmath_number_obj_t*)((char*)(obj) - XtOffsetOf(bcmath_number_obj_t, std));
+}
+
+static zend_always_inline bcmath_number_obj_t *get_bcmath_number_from_zval(const zval *zv)
+{
+	return get_bcmath_number_from_obj(Z_OBJ_P(zv));
+}
+
+static zend_always_inline zend_string *bcmath_number_value_to_str(bcmath_number_obj_t *intern)
+{
+	if (intern->value == NULL) {
+		intern->value = bc_num2str_ex(intern->num, intern->scale);
+	}
+	return intern->value;
+}
+
+static zend_object *bcmath_number_create(zend_class_entry *ce)
+{
+	bcmath_number_obj_t *intern = zend_object_alloc(sizeof(bcmath_number_obj_t), ce);
+
+	zend_object_std_init(&intern->std, ce);
+	object_properties_init(&intern->std, ce);
+
+	intern->num = NULL;
+	intern->value = NULL;
```

Comment:
You don't need to explicitly set these to NULL as `zend_object_alloc` initializes all bytes to 0 already.

Reply:
Fixed in https://github.com/php/php-src/pull/13741/commits/56f5b82bc72f0e492a141ec230f40af8f8cdb7a1